### ISSUE TITLE
[Bug Report] Got an unexpected keyword argument 'image' exception deploying Model

### ISSUE THREAD

**Issue by eitansela** [aws/amazon-sagemaker-examples#2074]
**Link to the notebook**
https://github.com/aws/amazon-sagemaker-examples/blob/master/sagemaker_model_monitor/introduction/SageMaker-ModelMonitoring.ipynb

**Describe the bug**
When deploying the model to Amazon SageMaker:

`model = Model(image=image_uri, model_data=model_url, role=role)`

Get the following exception:

```
TypeError                                 Traceback (most recent call last)
<ipython-input-5-8d04be9e6924> in <module>
      7 image_uri = get_image_uri(boto3.Session().region_name, 'xgboost', '0.90-1')
      8 
----> 9 model = Model(image=image_uri, model_data=model_url, role=role)

TypeError: __init__() got an unexpected keyword argument 'image'

```

**To Reproduce**
Run the ModelMonitoring.ipynb notebook cell by cell until getting to _**Deploy the model to Amazon SageMaker**_ cell.

**Comment by eitansela** [comment]
Got another exception when executing _**Invoking the deployed model**_ cell.

On the following line:
`predictor = RealTimePredictor(endpoint=endpoint_name,content_type='text/csv')`

Got exception:
```
TypeError                                 Traceback (most recent call last)
<ipython-input-6-8cf73404297d> in <module>
      2 import time
      3 
----> 4 predictor = RealTimePredictor(endpoint=endpoint_name,content_type='text/csv')
      5 
      6 # get a subset of test data for a quick test

/opt/conda/lib/python3.7/site-packages/sagemaker/deprecations.py in __init__(self, *args, **kwargs)
    171             """Provides a warning for the class name."""
    172             renamed_warning(f"The class {name}")
--> 173             super(DeprecatedClass, self).__init__(*args, **kwargs)
    174 
    175     return DeprecatedClass

TypeError: __init__() missing 1 required positional argument: 'endpoint_name'
```

**Comment by eitansela** [comment]
Got another exception when executing **_View captured data_** cell.

On the following cell:
```
s3_client = boto3.Session().client('s3')
current_endpoint_capture_prefix = '{}/{}'.format(data_capture_prefix, endpoint_name)
result = s3_client.list_objects(Bucket=bucket, Prefix=current_endpoint_capture_prefix)
capture_files = [capture_file.get("Key") for capture_file in result.get('Contents')]
print("Found Capture Files:")
print("\n ".join(capture_files))
```

Got exception:
```
TypeError                                 Traceback (most recent call last)
<ipython-input-7-9dc354b11722> in <module>
      2 current_endpoint_capture_prefix = '{}/{}'.format(data_capture_prefix, endpoint_name)
      3 result = s3_client.list_objects(Bucket=bucket, Prefix=current_endpoint_capture_prefix)
----> 4 capture_files = [capture_file.get("Key") for capture_file in result.get('Contents')]
      5 print("Found Capture Files:")
      6 print("\n ".join(capture_files))

TypeError: 'NoneType' object is not iterable
```

This is because the delay is 0.5 seond X 120 = 60 seconds, while the delivery of capture data to Amazon S3 can require a couple of minutes.

**Comment by aaronmarkham** [comment]
Fixed with #2075
